website/docs/components/drag-and-drop/drag-and-drop.md

```diff
@@ -9,117 +9,59 @@ tabs: Design('drag-and-drop'), A11y('drag-and-drop-a11y'), API('drag-and-drop-ap
 **Drag and drop** is a component that allows user to move elements within the interface, changing their order and sometimes their properties.
 
 ::: tip
-One common use case is reordering columns in a table, using dropdowns known as "Table settings controls".
+One common use for drag and drop is a dropdown with a list of table columns that can be reordered, usually titled **Table settings** or **Manage columns**.
 :::
 
-The component imitates the physical dragging of objects, and two essential considerations for this are:
+The component imitates physically dragging objects, and two essential considerations for this are:
 
-- Clearly indicating which interface elements can be dragged and where.
-- Providing visual cues during dragging to show that the element is being moved and where it can be placed.
+- Clearly indicate which interface elements can be dragged and where.
+- Provide visual cues during dragging to show that the element is being moved and where it can be placed.
 
 ### Component composition
 
-`DnD` is a container that wraps the necessary area for drag and drop functionality. The `DropZone` is where the element can be dropped, and the `Draggable` is the element you can move.
+The component consists of the following parts:
 
-![](static/dragdrop-scheme.png)
-
-## Appearance
-
-### Icon for identification of the possibility to drag the object
-
-When hovering over a draggable element, it's recommended to display the `MoveAlt` icon, which should use the `--icon-secondary-neutral` token for its color.
-
-![](static/drag-hover.png)
+1. `DragAndDrop` is a container for `Draggable` elements and related `DropZone` elements.
+2. `DragAndDrop.DropZone` is where the element can be dropped.
+3. `DragAndDrop.Draggable` is the element you can move.
 
-### Shadow of the object which can be dragged
-
-`--box-shadow-dnd` token is used for the shadow.
+![](static/dragdrop-scheme.png)
 
-![](static/dragging.png)
+## Grabbing
 
-## Area for dragging
+| State description | Illustration |
+|-------------------|--------------|
+| Drag indicators are hidden in the default state.  | ![](static/drag-default.png)  |
+| Drag indicator appears when the draggable object is hovered or focused.  | ![](static/drag-hover.png)  |
+| When the grab area is hovered, mouse cursor turns into `grab`.  | ![](static/hover-grab.png)  |
+| When the object is grabbed, mouse cursor turns into `grabbed`, the object box-shadow changes to `--box-shadow-dnd`.  | ![](static/dragging.png)  |
 
-The area for dragging is where you can click and hold the mouse or use the keyboard to move a large object, like a card or table row. We recommend using the dotted pattern that appears when user hovers over the area or use the `Tab` key to navigate to the element. This pattern can be placed both horizontally, as in the case of cards, and vertically, as in table rows.
+Drag indicator can be placed vertically, as in lists, or horizontally, as in cards.
```

Comment:
Just a proposal:

Maybe to rephrase the last part to something like:
"<...> as in the list of cards"?

Reply:
oh I got it
but I'd rather replace "in lists" with "in list items", to make the phrase internally consistent